### DISCUSSION TITLE
Unable to unset tailwindcss variable namespaces

### DISCUSSION THREAD

**Discussion by Stephan-MC** [tailwindlabs/tailwindcss#15522] [Help] [answered]
i can't reset tailwindcss default css variables. example:
```css
--breakpoint-*: initial;
```
and anyother tailwindcss variables i try to reset throws the following error:

![image](https://github.com/user-attachments/assets/6012d5cb-be0b-4fec-8aa4-6ffd507b2327)

Personally, i think it should be a sass related issue. haven't tried in css files yet.

**Comment by wongjn** [comment, marked as answer]
Yes, seems like a Sass issue trying to parse the special Tailwind syntax. Try using string interpolation:

```scss
@theme {
  #{'--breakpoint-*'}: initial;
}
```
Seems to work [in the Sass Playground](https://sass-lang.com/playground/#eJwzNHQoyUjNTVWo5lJQUK5W19VNKkpNzC7Iz8wr0dVSr7VSyMzLLMlMzLHmqgUALQIN6g==?s=L1C1-L3C2).

**Reply by Stephan-MC** [reply]
thanks. let me go give it a try

**Reply by Stephan-MC** [reply]
It worked. :+1:  thanks...

but my IDE is now full of errors :cry: . Even though it doesn't have compilation issues.

i also realised the following below works fine
```scss
@theme {
    --breakpoint-#{'*'}: initial;
}
```

**Comment by Stephan-MC** [comment]
@wongjn  please how can i fix this too in sass. 

```scss
@utility scrollbar-hidden {
  &::-webkit-scrollbar {
    display: none;
  }
}
```
